src/python/deepgnn/graph_engine/snark/convert.py

```diff
@@ -118,17 +118,31 @@ def convert(self):
                 num_workers=self.worker_count,
             )
 
-        for _, data in enumerate(self.file_iterator):
-            for line in data:
-                d.dispatch(line)
-
-        d.join()
+        if self.worker_count >= 1:
```

Comment:
I think we debug by using threads instead of processes, so exceptions can propagate to the main thread.

Reply:
use_threads is enabled by default and no errors are propogated.